src/shared/components/Header/index.tsx

```diff
@@ -11,6 +13,32 @@ interface InformationContentProps {
   githubIssueURL: string;
 }
 
+const VersionInfo = () => {
+  const preloadedState: RootState = (window as any).__PRELOADED_STATE__;
```

Comment:
You should not access the `__PRELOADED_STATE__` directly in a component. In fact you should never do this anywhere apart from when you bootstrap your app and you want to dehydrate the global state with the server-side computed state. 

If you need to connect part of your state to a component, use the `react-redux` library and the `connect` higher order function, and finally you need to make sure you do this in a `container` and not a `component`